src/System.Device.Gpio/System/Device/Pwm/Channels/UnixPwmChannel.Linux.cs

```diff
@@ -34,11 +34,11 @@ internal class UnixPwmChannel : PwmChannel
             int frequency = 400,
             double dutyCyclePercentage = 0.5)
         {
-            _chipPath = $"/sys/class/pwm/pwmchip{_chip}";
-            _channelPath = $"{_chipPath}/pwm{_channel}";
             _chip = chip;
             _channel = channel;
-            Validate();
+            _chipPath = $"/sys/class/pwm/pwmchip{_chip}";
+            _channelPath = $"{_chipPath}/pwm{_channel}";
+			Validate();
```

Comment:
nit: off by space

Reply:
Yeah, my IDE is set to tabs, I think I solved this in the second commit ... Thanks for the review